utils/wasm-builder/src/optimize.rs

```diff
@@ -123,3 +129,22 @@ pub fn do_optimization(
     }
     Ok(())
 }
+
+pub fn check_exports(module: &Module, path: &Path) -> Result<()> {
+    if module
+        .export_section()
+        .ok_or_else(|| {
+            crate::builder_error::BuilderError::ExportSectionNotFound(path.to_path_buf())
+        })?
+        .entries()
+        .iter()
+        .any(|entry| {
+            matches!(entry.field(), "init" | "handle")
+                && matches!(entry.internal(), Internal::Function(_))
+        })
+    {
+        Ok(())
+    } else {
+        Err(crate::builder_error::BuilderError::RequiredExportFnNotFound(path.to_path_buf()).into())
```

Comment:
We can import it to be shorter: 
```suggestion
        Err(BuilderError::RequiredExportFnNotFound(path.to_path_buf()).into())
```